root/plex-common.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+CONT_CONF_FILE="/version.txt"
+
+function addVarToConf {
+  local variable="$1"
+  local value="$2"
+  if [ ! -z "${variable}" ]; then
+    echo ${variable}=${value} >> ${CONT_CONF_FILE}
+  fi
+}
+
+function readVarFromConf {
+  local variable="$1"
+  declare -n value=$2
+  if [ ! -z "${variable}" ]; then
+    value="$(grep -w ${variable} ${CONT_CONF_FILE} | cut -d'=' -f2 | tail -n 1)"
```

Comment:
## Explanations for reading value :

Expecting file should be like:
```
version=beta
plex-distro=debian
...
```

Value is : look for variable name (grep), then take the part after the `'='` sign (cut), then take the last value if multiple (tail) (that should not happen but in case).